libcloudforensics/gcp.py

```diff
@@ -718,38 +711,37 @@ def add_labels(self, new_labels_dict, blocking_call=False):
     """
 
     get_operation = self.get_operation().execute()
-    labelFingerprint = get_operation['labelFingerprint']
+    label_fingerprint = get_operation['label_fingerprint']
```

Comment:
I think this will break things: https://cloud.google.com/compute/docs/labeling-resources

Reply:
Fixed